internal/dependencymanager/dependencyinstaller.go

```diff
@@ -308,6 +318,30 @@ func (di *DependencyInstaller) updateDependencyDeployment(contractName string) e
 	return nil
 }
 
+func (di *DependencyInstaller) updateDependencyAlias(contractName, aliasNetwork string) error {
+	var missingNetwork string
+
+	if aliasNetwork == "testnet" {
+		missingNetwork = "mainnet"
+	} else {
+		missingNetwork = "testnet"
+	}
+
+	label := fmt.Sprintf("Enter an alias address for %s on %s if you have one, otherwise leave blank", contractName, missingNetwork)
+	raw := util.AddressPromptOrEmpty(label, "Invalid alias address")
+
+	if raw != "" {
+		contract, err := di.State.Contracts().ByName(contractName)
+		if err != nil {
+			return err
+		}
+
+		contract.Aliases.Add(missingNetwork, flowsdk.HexToAddress(raw))
```

Comment:
Does this alias get saved?  It looks like to me we save in `updateDependencyState` & `updateDependencyDeployment`, so I don't think the state gets saved from what I can see after this alias is added.

Also on that note -- should we be saving at every step, or just once at the end.  IMO we should probably not be saving at intermediate steps because there is no guarantee that the user will not abort/error will not be encountered at a later point.  I think the expected behaviour if the command does not successfully complete is probably that the user's `flow.json` configuration did not change